### PR TITLE
chore: (Docker) Use env variables in the nginx config for port and domain

### PR DESCRIPTION
**Pull request by 7mochi**
# Describe your changes

## Checklist
- [x] I've manually tested my code
- [x] The changes pass pre-commit checks (`make lint`)
- [x] The changes follow coding style

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -57,13 +57,15 @@ services:
     depends_on:
       - bancho
     volumes:
-      - ./ext/nginx.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./ext/docker-nginx.conf.example:/etc/nginx/templates/nginx.conf.template:ro
```
Comment (NiceAesth) [inline docker-compose.yml:60]: very odd choice to use an "example" file in here, when the .env file is with the normal name

File: .github/docs/wiki/Setting-up-(Docker).md [hunk 2]
```diff
@@ -39,17 +39,6 @@ cp .env.example .env
 nano .env
 ```
 
-## configuring a reverse proxy (we'll use nginx)
```
Comment (NiceAesth) [inline .github/docs/wiki/Setting-up-(Docker).md:42]: these changes should also appear in the translations

**Review by NiceAesth — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (7mochi) [reply docker-compose.yml:60]: the filename was changed in #425 so I used the file as is

Comment (NiceAesth) [reply docker-compose.yml:60]: That PR is a bit problematic

# create a configuration file from the sample provided
cp .env.example .env

this is now incorrect in both setup methods (+ translations)

also, the example cannot remain in the compose. i propose adding instructions to do something similar to .env, or to just rename the file and move it in `docker/` (could be the better solution)

**Review by NiceAesth — APPROVED** [review]

**Pull request merged**